staging/src/k8s.io/apimachinery/pkg/util/httpstream/httpstream.go

```diff
@@ -78,6 +78,8 @@ type Connection interface {
 	// SetIdleTimeout sets the amount of time the connection may remain idle before
 	// it is automatically closed.
 	SetIdleTimeout(timeout time.Duration)
+	// RemoveStreams can be used to remove a set of streams from the Connection.
+	RemoveStreams(identifiers []uint32)
```

Comment:
Why not take a list of Streams?

Reply:
Changing it to `httpstream.Stream` would result in an import cycle:
```go
	// RemoveStreams can be used to remove a set of streams from the Connection.
	RemoveStreams(identifiers []httpstream.Stream)

```
```
package k8s.io/kubernetes/cmd/kube-apiserver
        imports k8s.io/kubernetes/cmd/kube-apiserver/app
        imports k8s.io/kube-aggregator/pkg/apiserver
        imports k8s.io/apimachinery/pkg/util/httpstream
        imports k8s.io/apimachinery/pkg/util/httpstream: import cycle not allowed
```

Reply:
I meant the Stream returned by line 73 above?

Reply:
Misunderstood that point, sorry. Changed as suggested :+1: